src/genie/libs/parser/iosxe/show_bootflash.py

```diff
@@ -0,0 +1,61 @@
+''' show_bootflash.py
+
+IOSXE parsers for the following show commands:
+    * show bootflash:
+'''
+
+# Python
+
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+# parser utils
+from genie.libs.parser.utils.common import Common
+
+# =================
+# Schema for:
+#  * 'show bootflash:'
+# =================
+
+class ShowBootflashSchema(MetaParser):
+    """Schema for show bootflash:."""
+    schema = {
+        'available': str,
+        'used': str
+    }
+
+# =================
+# Parser for:
+#  * 'show bootflash:'
+# =================
+class ShowBootflash(ShowBootflashSchema):
+    """Parser for show bootflash:"""
+
+    cli_command = 'show bootflash:'
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        #         available used
```

Comment:
Could you please put a complete captured line example above the regex. Based on your golden output, something like;
`13755338752 bytes available (489017344 bytes used)`